libs/providers/go-feature-flag-web/tsconfig.lib.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "lib": ["ES2015", "DOM"],
```

Comment:
I wonder if we need new generators for client components. I guess you added `DOM` manually?

Reply:
Yes, I copy/paste it from the `flagd-web` provider, but yes I agree we need a generator for web provider.